account_credit_control/views/credit_control_line.xml

```diff
@@ -210,23 +210,16 @@
                 decoration-danger="state in ('error', 'email_error')"
                 decoration-muted="state == 'ignored'"
                 decoration-success="state == 'sent'"
+                open_form_view="True"
             >
-                <button
-                    name="button_credit_control_line_form"
-                    aria-label="Control Credit Line Form"
-                    title="Control Credit Line Form"
-                    type="object"
-                    icon="fa-external-link"
-                    class="oe_stat_button"
-                />
-                <field name="date" />
-                <field name="date_due" />
+                <field name="date" readonly="state != 'draft'" />
+                <field name="date_due" readonly="state != 'draft'" />
                 <field name="level" />
-                <field name="manually_overridden" />
+                <field name="manually_overridden" readonly="state != 'draft'" />
                 <field name="state" />
-                <field name="channel" />
+                <field name="channel" readonly="state != 'draft'" />
                 <field name="invoice_id" />
-                <field name="partner_id" />
+                <field name="partner_id" readonly="true" />
```

Comment:
```suggestion
                <field name="partner_id" readonly="True" />
```
It probably works correctly, but the usual way is to use it with a capital T or `readonly="1"`.